testing/split-view.tsx

```diff
@@ -4,34 +4,36 @@ import {HeadingMedium} from "@khanacademy/wonder-blocks-typography";
 import * as React from "react";
 import ReactJson from "react-json-view";
 
-import {interactiveSizes} from "../packages/perseus/src/styles/constants";
-
 type Props = {
-    leftTitle: React.ReactNode;
-    left: React.ReactNode;
-    rightTitle?: string;
+    rendererTitle: React.ReactNode;
+    renderer: React.ReactNode;
+    JSONTitle?: string;
     jsonObject: any;
 };
 
-const SideBySide = ({
-    leftTitle = "Renderer",
-    left,
-    rightTitle = "Perseus JSON",
+/**
+ * This component is used in Storybook to render both the renderer and the JSON object in a split view.
+ * Currently, the renderer is displayed above the JSON object.
+ */
+const SplitView = ({
```

Comment:
There was some discussion about how the team would like to restructure the Storybook views. By moving the PerseusJSON view below the Renderer (Widgets), we can give both elements more room to breathe. This is particularly noticeable on the Number Line Widget, or with the Graded Group JSON. 

As a result of this move, "SideBySide" no longer felt like an accurate name for this component. I've renamed it to "SplitView" to be more generic, just in case we ever wish to reverse this decision. I've also updated variables accordingly.